diff --git a/update.php b/update.php
index eb8fe98..3697ad5 100644
--- a/update.php
+++ b/update.php
@@ -104,7 +104,14 @@ function update_script_selection_form($form, &$form_state) {
 
   if (empty($count)) {
     drupal_set_message(t('No pending updates.'));
-    unset($form);
+    if ($incompatible_updates_exist) {
+      // Work around issue with collapsible regions
+      $form['start']['#collapsed'] = FALSE;
+      $form['start']['#collapsible'] = FALSE;
+    }
+    else {
+      unset($form);
+    }
     $form['links'] = array(
       '#markup' => theme('item_list', array('items' => update_helpful_links())),
     );
